docs/markdown/Wrap-dependency-system-manual.md

```diff
@@ -313,6 +314,26 @@ method = cmake
 [provide]
 foo-bar-1.0 = foo_bar_dep
 ```
+### Cargo wraps
+
+Cargo subprojects automatically override the `<package_name>-rs` dependency name.
+`package_name` is defined in `[package] name = ...` section of the `Cargo.toml`
+and `-rs` suffix is added. That means the `.wrap` file should have
+`dependency_names = foo-rs` in their `[provide]` section when `Cargo.toml` has
+package name `foo`.
+
+Cargo subprojects require a toml parser. Python >= 3.11 have one built-in, older
+Python versions require either the external `tomli` module or `toml2json` program.
+
+For example, a Cargo project with the package name `foo-bar` would have a wrap
+file like that:
+```ini
+[wrap-file]
+...
+method = cargo
+[provide]
+dependency_names = foo-bar-rs
+```
```

Comment:
If I interpret the test added below correctly, the `[provide]` section seems to be optional as it is not in the test wrap files. I think this is a great convenience. Maybe it should be highlighted here?

Reply:
There is always an implicit `dependency_names = anything` for wrap file named `anything.wrap`. That's not specific to rust, but that means if you have `foo-bar-rs.wrap` you don't need explicit `dependency_names = foo-bar-rs`.